extensions/bit-manipulation-1/README.md

```diff
@@ -0,0 +1,48 @@
+# General design
+
+**Extension State: Under Development**  
+**Requires: Base, VWI, CP2.0**  
+**CPUID Bit: CP2.4**
+
+# Overview
+
+This extension adds rotate with carry and various bit operations.
+
+# Added Instructions
+
+These instructions are in the expanded calculation opcode section of instructions.
+
+### Opcode table
+
+| `C CCCC CCCC` | NAME      | Operation                                              | Flags | Comment     |
+|---------------|-----------|--------------------------------------------------------|-------|-------------|
+| `0 0000 1000` | `RCL`     | <code>C:A ← (C:A << B) &#124; (C:A >> -B)</code>       | `ZNC` | (1) (2) (3) |
+| `0 0000 1001` | `RCR`     | <code>A:C ← (A:C >> B) &#124; (A:C << -B)</code>       | `ZNC` | (1) (2) (3) |
+| `0 0000 1010` | `POPCNT`  | Counts the number of set bits in the `B` input         | `ZN`  | (4)         |
+| `0 0000 1011` | `BITSWAP` | Swaps all the bits based on the `SS` bits              | `ZN`  | (5)         |
+| `0 0000 1100` | `CTZ`     | Counts the number of `0` bits before the first `1` bit | `ZN`  |             |
+| `0 0000 1101` | `CLZ`     | Counts the number of `0` bits after the last `1` bit   | `ZN`  |             |
+| `0 0000 1110` | `NOT`     | <code>A ← ~B</code>                                    | `ZN`  |             |
+| `0 0000 1111` | `ANDN`    | <code>A ← ~A &#38 B</code>                             | `ZN`  |             |
+| `0 0001 1000` | `LSB`     | <code>A ← B &#38 -B</code>                             | `ZN`  |             |
+| `0 0001 1001` | `LSMSK`   | <code>A ← B ^ (B - 1)</code>                           | `ZN`  |             |
+| `0 0001 1010` | `RLSB`    | <code>A ← B &#38 (B - 1)</code>                        | `ZN`  |             |
+| `0 0001 1011` | `ZHIB`    | <code>A ← A &#38 ((1 << B) - 1)</code>                 | `ZN`  |             |
```

Comment:
These http escapes aren't correct, the `;` is missing, see the rich preview on github.
```suggestion
| `0 0000 1111` | `ANDN`    | <code>A ← ~A &#38; B</code>                             | `ZN`  |             |
| `0 0001 1000` | `LSB`     | <code>A ← B &#38; -B</code>                             | `ZN`  |             |
| `0 0001 1001` | `LSMSK`   | <code>A ← B ^ (B - 1)</code>                           | `ZN`  |             |
| `0 0001 1010` | `RLSB`    | <code>A ← B &#38; (B - 1)</code>                        | `ZN`  |             |
| `0 0001 1011` | `ZHIB`    | <code>A ← A &#38; ((1 << B) - 1)</code>                 | `ZN`  |             |
```